Ginger/Ginger/UserControlsLib/UCListView/UcListView.xaml.cs

```diff
@@ -137,27 +138,35 @@ public IObservableList DataSourceList
                     }
 
                     mObjList = value;
-                    this.Dispatcher.Invoke(() =>
+
+                    xSearchTextBox.Text = "";
```

Comment:
why xSearchTextBox.Text = ""; was pulled out from .Dispatcher.Invoke(() this may cause issues in case diffrent thread will set the list DataSource because here we modify UI control- i will merge the code but i belive that will cause issue